src/firefox/lib/glue.js

```diff
@@ -129,11 +138,12 @@ function setUpConnection(freedom, panel, button) {
   };
 
   /* Allow pages in the addon and uproxy.org to send messages to the UI or the core */
+  var contentScriptFile = self.data.url('scripts/content-proxy.js');
```

Comment:
I'd name this `contentProxyFile` (since while it is a content script, it's more importantly a specific script)